### DISCUSSION TITLE
Clone Deepin only opens via terminal

### DISCUSSION THREAD

**Discussion by serial02** [linuxdeepin/developer-center#3037] [Q&A | 问答板块] [answered]
Hello Clone Deepin Team

Could you help us with this?

https://arcolinuxforum.com/viewtopic.php?f=117&t=1314&sid=3b069db6fb8dc07e2159eb6995f076f6

Despite the constant updates it does not open in the Deepin Icons menu?
What can be done so that Deepin Clone can also be opened in the Deepin Icons menu?

![JXHcfRJ](https://user-images.githubusercontent.com/32621790/61594976-d57b9b00-abc7-11e9-974d-4ddb48e11c89.png)

A aplicação do clone só é aberta através do terminal, como mostrado abaixo. 

![ENl44P9](https://user-images.githubusercontent.com/32621790/61594985-f0e6a600-abc7-11e9-8a6b-da0c31fe7390.png)

Thank you in advance for your usual attention

Serial

**Comment by BLumia** [comment]
cc @zccrs

**Comment by serial02** [comment]
Four months have passed and the same situation remains. Is there any predictability of when this error will be corrected?

Passaram-se quatro meses e continua a mesma situação. Existe previsibilidade de quando este erro será corrigido?

**Comment by BLumia** [comment]
It seems no one is maintaining `deepin-clone` project at the moment. I'm afraid we need to wait longer and see if anyone is picking up this project :/

**Comment by marcelosenabr** [comment, marked as answer]
Hello.

From what I realized the @serial02, ran the deepin-clone without root privileges, which is required for the application to run. hence the error. and the fact that it does not appear on the menu, I believe it is intentional on the part of the deepin developer team.

I created a toturial and a video showing the error and how to solve it:
https://marcelosena.com/corrigindo-erro-do-deepin-clone-nao-aparecer-no-menu-de-atalhos/

Video: https://youtu.be/sYFFeakdSfM

And 

https://github.com/linuxdeepin/deepin-clone/issues/16
